src/App.js

```diff
@@ -1,88 +1,156 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
+import { ToDoList } from './components/ToDoList';
+import { Footer } from './components/Footer';
+import { Context } from './context';
+
+export function App() {
+  const [listOfToDos, setListOfToDos] = useState([]);
+  const [toDosToShow, setToDosToShow] = useState(listOfToDos);
+  const [notCompletedToDos, setNotCompletedToDos] = useState([]);
+  const [newToDoTitle, setNewToDoTitle] = useState('');
+  const [statusToShow, setStatusToShow] = useState('all');
+
+  useEffect(() => {
+    setNotCompletedToDos(
+      listOfToDos.filter(todo => todo.completed === false),
+    );
+
+    setToDosToShow(listOfToDos);
+  }, [listOfToDos]);
+
+  function addTodo(event) {
+    if (newToDoTitle.trim().length > 0) {
+      setListOfToDos(prevState => [
+        ...listOfToDos,
+        { title: newToDoTitle, id: +new Date(), completed: false },
+      ]);
```

Comment:
no need to use functional setState here, by the way, I noticed that you don't even use it here